Homework_5_6/Homework_5_6.md

```diff
@@ -0,0 +1,80 @@
+## Task 1
+
+### SCRUM CEREMONIES:
+
+- **Product backlog refinement** - it is the meeting before sprints where you take tasks from the backlog and you refine/specify details that are needed, between team and product owner.
+- **Sprint planning** - it is a meeting before sprint, where the goals of sprint and how goals are delivered are put in place.
+- **Daily scrum** - it is a meeting where the team gets together every day and every member says what they did yesterday, what they do that day and if there are any blockers.
+- **Sprint review** - meeting where the team comes together and sees the end-work/results and see if there are any changes needed.
+- **Sprint retrospective** - meeting where the team discusses what went great and what can be improved next time.
+
+### SCRUM ROLES:
+
+- **Scrum Master** - is a development team's team lead/supporter. Scrum Master initiates meetings, coaches, supports and protects the team, communicates, helps with blockers, reports. Scrum Master is between PM and the management.
+- **Product Owner** - responsible for the final product, takes feedback from different stakeholders and translates technical and business sides to each other.
+- **Development Team** - is a group of people who do the activities (planning, programming, testing, analyzing) to reach the needed goal.
+
+## Task 2
+
+### Epic: Yoga booking system
+
+### Stories: 
+- S1 As a yoga class client, I want to see what classes I could take(Classes display)
+  - Task 1.1 - Design UI/UX
+  - Task 1.2 - Write front-end for booking and classes info (at the same page)
+  - Task 1.3 - Connect API to show user-booking and classes info
+- S2 As a yoga class client, I want to book new classes (information storing in SQL and booking)
+  - Task 2.1 - Design DB architecture
+  - Task 2.2 - DB creating
+- S3 As a yoga class client, I want to change my information or update my participation (back-end)
+  - Task 3.1 - Mapping end-points
+  - Task 3.2 - Implementing end-point
+    - 3.2.1 - User-booking API
+    - 3.2.2 - Classes info API
+
+- S1 tasks 1.1 and 1.2 should be done in order, because you have to have design before you start coding.
+- S2 tasks should be done in order, because you can't create database before the design.
+- S3 tasks 3.1 and 3.2 should be done in particular order, because you can't implement before mapping, but implementing them can be done in parallel.
+- S2, S3 could be also be done parallel and S1 can be done partly in parallel, because you can write front-end with mocked API response when endpoints are not ready. 
+- Task 1.1, 2.1, 3.1 - can be done in parallel and 2.1 and 3.1 should be done by system architect(s), to see how everything in the application goes together.
+
+## Task 3
```

Comment:
10/10 great job on explaining the cinema design. I like the design as you consider the edge cases in a payment like using a gift card, and third-party payments.